metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -135,6 +141,7 @@ entities:
       - browsePaths # unclear if this will be used
       - status
       - domains
+      - containerPath
```

Comment:
the containerPath for a container should not include itself, right?

for root level containers, should be emit an empty containerPath?

Reply:
that's right! that's what i would expect

Reply:
perfect!